tests/instruments/chroma/test_chroma62000p.py

```diff
@@ -0,0 +1,39 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2019 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import pytest
+from pymeasure.instruments.chroma.chroma62000p import Chroma62024P6008
+from time import sleep
+
+@pytest.mark.skip(reason="no way of currently testing this just in SW")
+def test_Chroma_current_limit_turn_on_and_off():
+    psu=Chroma62024P6008('USB0::0x1698::0x0837::007000000446::0::INSTR')
+    print(psu.id)
+    psu.output_current_limit = 1
+    psu.output_voltage_level = 5
+    psu.output_enable()
+    sleep(2)
+    print(psu.output_voltage)
```

Comment:
In addition to printing, this test should assert that the voltage level is reached (when a device is connected, of course): 
```
assert psu.output_voltage == pytest.approx(5)